Client/Frontend/Menu/MenuPresentationAnimator.swift

```diff
@@ -83,44 +83,30 @@ extension MenuPresentationAnimator {
         }
         menuView.frame = container.bounds
 
-        // Insert tab tray below the browser and force a layout so the collection view can get it's frame right
+        let bgView = UIView()
+        bgView.frame = container.bounds
+
         if presenting {
-            container.insertSubview(menuView, belowSubview: bottomView)
+            container.addSubview(menuView)
+            container.insertSubview(bgView, belowSubview: menuView)
             menuView.layoutSubviews()
         }
 
-        let vanishingPoint: CGPoint
-        if let sourceView = sourceView {
-            vanishingPoint = menuView.convert(sourceView.center, from: sourceView.superview)
-        } else {
-            vanishingPoint = CGPoint(x: menuView.center.x, y: menuView.frame.size.height)
-        }
-
-        let minimisedFrame = CGRect(origin: vanishingPoint, size: CGSize.zero)
-        guard let menuViewSnapshot = menuView.snapshotView(afterScreenUpdates: presenting) else {
-            transitionContext.completeTransition(true)
-            return
-        }
+        let vanishingPoint = CGPoint(x: menuView.frame.origin.x, y: menuView.frame.size.height)
```

Comment:
Disregard. I just saw your comment on this PR that this animation is slightly different :-)